tools/blaster.js

```diff
@@ -203,18 +204,20 @@ function createUser(index, callback) {
     } else {
         accountName = createUniqueName(accountNames);
     }
-    const params = {email, password: getPassword()};
+    const params = { email, password: getPassword() };
     client.createUser(accountName, userName, params, (err, data) => {
         if (!error && !err) {
             const acc = accounts[accountName];
             acc.usersNames.push(userName);
             acc.usersEmails.push(email);
-            acc.users[userName] = {email, data, accessKeys: [], secretKeys: {}};
+            acc.users[userName] = {
+                email, data, accessKeys: [], secretKeys: {},
```

Comment:
Make is more readable:
```suggestion
                email,
                data,
                accessKeys: [],
                secretKeys: {},
```